libvcx/src/aries/handlers/proof_presentation/prover/prover.rs

```diff
@@ -34,7 +34,8 @@ impl Prover {
 
     pub fn retrieve_credentials(&self) -> VcxResult<String> {
         trace!("Prover::retrieve_credentials >>>");
-        let presentation_request = self.prover_sm.presentation_request().request_presentations_attach.content()?;
+        let presentation_request = self.prover_sm.presentation_request().request_presentations_attach.content()?; // TODO: How does this have known size at compile time?
```

Comment:
Why wouldn't it? Seems like it's just a string?

Reply:
This was meant to be 'How doesnt this have ...', I left it due to incorrect static analysis